yellow.sdef

```diff
@@ -86,3 +86,6 @@ kernelModules:
     linux_kernel_modules/cypwifi/cypwifi
     linux_kernel_modules/cp2130/cp2130
 }
+
+// Uncomment to include support for the CAN IoT card
```

Comment:
Customers can build on top of the yellow.sdef by creating their own .sdef file that includes yellow.sdef and any other .sinc files or .sdef files they want.  So, I'm thinking we should reduce clutter in the yellow.sdef by not adding these commented out includes, but rather we can create a "CAN IoT Card How-To" document on mangoh.io that tells people to `#include` the .sinc (along with the other steps required to obtain one of these cards and to build and test the software support for this IoT card).

Reply:
Of course, I'm not asking you to write that doc now.  Maybe just put a comment at the top of the .sinc file for now, if there isn't already one.

Reply:
Agreed with Jen. We can build an iot card sinc file.